ETC Course: 2. The Cypherpunks

DonaldMcIntyre opened the pull request:
This is class 2 of the ETC Course that will be published on 2022-12-29. CN translation coming soon.

w1g0 commented:
@DonaldMcIntyre, maybe keep it in draft until the CN version is added?

DonaldMcIntyre commented:
> @DonaldMcIntyre, maybe keep it in draft until the CN version is added?

Yes, soory I forgot to move it to draft.

bobsummerwill commented:
Cypherpunks group formed 1992 as per NYT, not 80s.  Interesting article here:

https://www.nytimes.com/2018/12/21/obituaries/timothy-c-may-dead.html?smid=nytcore-ios-share&referringSource=articleShare

bobsummerwill commented:
https://www.coindesk.com/markets/2018/10/19/enough-with-the-ico-me-so-horny-get-rich-quick-lambo-crypto/ I had not seen before.

DonaldMcIntyre commented:
> Cypherpunks group formed 1992 as per NYT, not 80s.  Interesting article here:
> 
> https://www.nytimes.com/2018/12/21/obituaries/timothy-c-may-dead.html?smid=nytcore-ios-share&referringSource=articleShare 

Tim May told me he started it in 1988.

DonaldMcIntyre commented:
> https://www.coindesk.com/markets/2018/10/19/enough-with-the-ico-me-so-horny-get-rich-quick-lambo-crypto/ I had not seen before.

Yes. This was his style. He wrote that text as a disorganized stream of thoughts, then got tired and sent the text to Coindesk and told them to do what they wanted with it. He passed away two months after. Very sad.

DonaldMcIntyre commented:
@bobsummerwill, I see what you mean. Yes, the formal naming and activities started in the early 90s so I changed the wording to add "early 90s". I also added text to clarify that "the individuals who would later become the Cypherpunks" in the 1980s. So, this is more accurate. Thank you!

TheCrowbill commented on content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md:
```diff
@@ -0,0 +1,111 @@
+
+Mr. May was like a role model who provided counsel, motivation, and direction to the group.
+
+Like many Libertarians, his interests revolved around privacy, reducing arbitrary intervention on the internet, and he had a special disdain for the taxman.
```
```suggestion
Like many Libertarians, his interests revolved around privacy, reducing arbitrary intervention on the internet, and he had a special disdain for the tax man.
```

TheCrowbill commented on content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md:
```diff
@@ -0,0 +1,111 @@
+
+Adam Back is another Cypherpunk who in 1997 was working on a way to avoid spam in a peer-to-peer email system. If the internet were to be decentralized, then tech company email services should be avoided and each user should be able to prevent spam.
+
+His solution was a system called HashCash that used proof of work; the heart of the technology in Bitcoin, and the heart of the consensus mechanism that ETC uses today; as a way of blocking spammers.
```
```suggestion
As a way of blocking spammers, he designed a system using proof of work called HashCash. Proof of work is at the heart of the technology used in Bitcoin and is the consensus mechanism that ETC uses today.
```
Edited for clarity, to remove awkward punctuation and repeated wording.

TheCrowbill commented on content/blog/2022-12-29-ethereum-classic-course-2-the-cypherpunks/index.md:
```diff
@@ -0,0 +1,111 @@
+
+![The four technologies that form part of BTC and ETC.](./etc-course-pioneer-tech-btc-etc.png)
+
+In our next class we will explain the four precursor technologies invented by these Cypherpunks that eventually gave way and formed part of Bitcoin and what is now Ethereum Classic:
```
```suggestion
In our next class we will explain the four precursor technologies invented by these Cypherpunks that eventually gave way to and formed part of Bitcoin and what is now Ethereum Classic:
```

TheCrowbill reviewed (approved):
LGTM

bobsummerwill reviewed (approved):
LGTM.  Thanks for the tweaks.
